<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenDialog</class>
 <widget class="QDialog" name="OpenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	border-image: url(:/Live/Resources/Open.png);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget
{
	
	
}
</string>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>461</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget_2
{
	
	background-color: rgb(38, 40, 45);
}</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>15</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="Title_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>8</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 126, 24);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Close_pushButton">
       <property name="geometry">
        <rect>
         <x>442</x>
         <y>8</y>
         <width>10</width>
         <height>10</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Close_pushButton{
	
	border-image: url(:/Close/Resources/Close_Normal.png);
}
QPushButton#Close_pushButton:hover{
	
	border-image: url(:/Close/Resources/Close_Hover.png);
}
QPushButton#Close_pushButton:disabled{
	
	border-image: url(:/Close/Resources/Close_Disabled.png);
}
QPushButton#Close_pushButton:checked{
	
	border-image: url(:/Close/Resources/Close_checked.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="OK_pushButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>132</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#OK_pushButton
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButtonYellow.png);
	font: 75 12px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}

QPushButton#OK_pushButton:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButtonYellow_H.png);
}

QPushButton#OK_pushButton:checked
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButtonYellow_C.png);
}

</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>461</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(254, 161, 24);
font: 75 14px &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>文件地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="File_lineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width:  1px 40px 40px 40px ;
border-radius : 4px;
border-width: 1px;
border-style: none;
border:1px groove rgb(50,53,59);border-radius:4px;

font: 12px &quot;微软雅黑&quot;;
color: rgb(166, 166, 166);

background-color: rgb(34, 36, 40);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Browse_pushButton">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton
{
	font: 75 12px &quot;微软雅黑&quot;;
	color: rgb(240, 240, 240);
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButton.png);
}

QPushButton:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButton_H.png);
}

QPushButton:checked
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButton_C.png);
}
</string>
         </property>
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rdzbdemo.qrc"/>
 </resources>
 <connections/>
</ui>
